.travis.yml

```diff
@@ -1,19 +1,27 @@
 language: rust
+
+env:
+  global:
+    - secure: O14lnD45q3oyuMtKU7Jfy1e+LSs91UZPnNuTPT9IZNoHPgJM77mdK1bwu5SqSXpmDEaq3uQBBhIKEKxiOgogiRAGcALnjSLIe02qrw8kR4hhB7IsuCsfJA3rgnH5NrSxV2ja5wHUkTtYORIB30GX/SU4qNw8C+HeAcXDaxPRNJYuIkE41jW121PDgheS7Ti49ui5JOKss/FdpI8Ii5xXvNf9Z60MSJKPsM8yaj+RiqDccXrFuyppVE/Ezn95gdJh1Sr7BUaP521FOhOYl4bvMEV0pHHMwaPD0b2Ku0yJ+GrZiZ2YqTmsNiaa4hN2AAv38w46KfhQuvJg173cEGf6hCrrF+IOQvmGG977xSO7tci2l5idjIZ1xtXsLQ9Yp49DBxd1s2I20j+fHmCImH1cV4+hEnMTQLITzemS3MkHIxbEgcMyssjZD7FASNU1C3fbeeHEG3JPM/hHj0mlGlhv7T54gL5Q7Qggnj+xsE44yU18bP0K/G48QBLeXDSfZzeaCbrif9bALAVnUMhRMSuW9EfOHcu7juaBAj5mPxWfN4CoQgqBFPh+V+q6+WViV0/5djoKlrOy323hNTg+CXIqWn9RYIbxbyRMGXIU6Z78Sr6Yls+8tK1cpdXOzQ6CMeUJh2H264hFMM1ZGvlQ+VYqRYMCViuVW4Aofuqr1zfsnz0=
+    - secure: xm+WIGYVoYvjAlWOaoli0YZ4LMFBKd7pG0qno4zrDMGrz1AwLRK6dce0+ROvIdEcuOror2j4wryYgDRMLYgxzhXeMMVFkxZuB1MmeC8VHirf0jNHNNihWwnEyTDGtj5VXijD/ipo4jiskBZEsg/oW97iFUudjl/rNT2R0Gy420QgGV4p3m9jTjBBL6giIQUNK0g+5aIkCfb4U/4gcpFA5nZJgyfQtnKKmVnEPl26vbU+Ui7F/wqTK0a7AJT3RCaEddtu7E5DKNnk4/45e4//LKNEE8I2KGqWlqWXqFYs+Cno1trDOcKm0yiTIquvbZsrOuH7wN1il4qVcsdgQLps2kFahpJyQJbJJViBkbqnlwRJDnxtszbrxSGW6HiVMSSd8AOthwJd3n7lMkBDwrwG9Dx7MiNY/NIdRxufEuoOw/6tj31oBJhM1s+/Fhjzlz0j5R5spTtcaNg6Q0Gi+F283jjwvobIzvUUKKSt4I1UZLTQNSzxrWTJZ4kJBG1JiTxH7BwHMtxhiaZn5ajk5HfVXScmBknZIovsylj5UXp1d5IhXUNq4jTXTdybOurhQ1XLVz/hw64oOIQkuftaNHuQMe3oB69aWLWpS6+r3k1ZCT/WODbS3xlhp/MXO3WLD4WY1OWJ522t/YOUQX3j72bZADlQOcU/OWQ5LWNC3XGibrk=
+
 cache: cargo
 
+services:
+  - docker
```

Comment:
I think it's better to use `docker` service only for jobs which really need them, not for all.